.appveyor.yml

```diff
@@ -58,19 +58,19 @@ install:
           Add-Content php.ini "`n extension=php_curl.dll"
           Add-Content php.ini "`n curl.cainfo=C:\tools\cacert\bundle.pem"
 
-          # Use a preview version of the extension that supports PHP 7.4
-          $DLLVersion = "5.7.1preview"
+          # Use a preview version of the extension that supports PHP 8.0
+          $DLLVersion = "5.9.0"
```

Comment:
This is no longer a preview version. It could be upgraded in `2.13.x`.

Reply:
Maybe I'll just use that code from the 2.13.x branch then?

https://github.com/doctrine/dbal/blob/7d00842301622a6890fd574676ddbbf7ee5f074d/.appveyor.yml#L66-L67

Reply:
That’s even better.